internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixStatusMessageChannel.java

```diff
@@ -171,7 +171,7 @@ public void sendToStorageNodes(String clusterName, StatusMessage message, String
       }
     }
     if (isSuccessful) {
-      LOGGER.info(numMsgSent + " messages have been send and processed. Message: {}.", message.toString());
+      LOGGER.info("{} messages have been send and processed. Message: {}.", numMsgSent, message.toString());
```

Comment:
remove toString

Reply:
Sure, let me check all this toString usages in this change.

Reply:
Thank you